tests/dependency_management/test_setup_py_codemod.py

```diff
@@ -0,0 +1,218 @@
+        before = """
+        from setuptools import setup
+        setup(
+            name="test pkg",
+            description="testing",
+            long_description="...",
+            author="Pixee",
+            packages=find_packages("src"),
+            package_dir={"": "src"},
+            python_requires=">3.6",
+            install_requires=[
+                "protobuf>=3.12,<3.18; python_version < '3'",
+                "protobuf>=3.12,<4; python_version >= '3'",
+                "psutil>=5.7,<6",
+                "requests>=2.4.2,<3"
+            ],
+            entry_points={},
+        )
+        """
+        tmp_file_path = Path(tmpdir / "not-setup.py")
+        self.run_and_assert_filepath(tmpdir, tmp_file_path, before, before)
+
+    def test_setup_call_no_install_requires(self, tmpdir):
+        before = """
+        from setuptools import setup
+        setup(
+            name="test pkg",
+            description="testing",
+            long_description="...",
+            author="Pixee",
+            packages=find_packages("src"),
+            package_dir={"": "src"},
+            python_requires=">3.6",
+        )
+        """
+        tmp_file_path = Path(tmpdir / "setup.py")
+        self.run_and_assert_filepath(tmpdir, tmp_file_path, before, before)
+
+    def test_setup_no_existing_requirements(self, tmpdir):
+        before = """
+        from setuptools import setup
+        setup(
+            name="test pkg",
+            description="testing",
+            long_description="...",
+            author="Pixee",
+            packages=find_packages("src"),
+            package_dir={"": "src"},
+            python_requires=">3.6",
+            install_requires=[],
+            entry_points={},
+        )
+        """
+
+        after = """
+        from setuptools import setup
+        setup(
+            name="test pkg",
+            description="testing",
+            long_description="...",
+            author="Pixee",
+            packages=find_packages("src"),
+            package_dir={"": "src"},
+            python_requires=">3.6",
+            install_requires=["defusedxml==0.7.1", "security~=1.2.0"],
+            entry_points={},
+        )
+        """
+        tmp_file_path = Path(tmpdir / "setup.py")
+        self.run_and_assert_filepath(tmpdir, tmp_file_path, before, after)
+
+    def test_setup_call_bad_install_requires(self, tmpdir):
+        before = """
+        from setuptools import setup
+        setup(
+            name="test pkg",
+            description="testing",
+            long_description="...",
+            author="Pixee",
+            packages=find_packages("src"),
+            package_dir={"": "src"},
+            python_requires=">3.6",
+            install_requires="some-package",
+        )
+        """
+        tmp_file_path = Path(tmpdir / "setup.py")
+        self.run_and_assert_filepath(tmpdir, tmp_file_path, before, before)
+
+    @pytest.mark.skip("Need to add support.")
```

Comment:
Obviously not for this ticket but I think this will definitely be possible for the simple cases.